scripts/build_lib.sh

```diff
@@ -4,9 +4,11 @@ set -e
 # always run from top of repo
 cd $(dirname $0)/..
 
+
+
 # this needs bash
 case "$OSTYPE" in
-  darwin*)  libname=lib/libpestutils.dylib ;;
+  darwin*)  libname=lib/libpestutils.dylib && export LDFLAGS="$LDFLAGS -Wl,-ld_classic";;
```

Comment:
I see this is a workaround Xcode 15 [documented here](https://developer.apple.com/documentation/xcode-release-notes/xcode-15-release-notes#Linking), which is deprecated for Xcode 16 [as documented here](https://developer.apple.com/documentation/xcode-release-notes/xcode-16-release-notes#Linking). Is this flag needed on CI? I'm wondering it it's better handled within meson, but I'd need a machine to test ideas with.

Reply:
I understand it now, and have boiled down the [logic here](https://github.com/sagemath/sage/blob/dc99dc846c5ebebddec75c4eb184d8cccf4996a5/src/bin/sage-env#L387-L395) into a shorter form.